sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -35,4 +35,12 @@ endif::[]
 include::modules/sandboxed-containers-common-terms.adoc[leveloffset=+1]
 include::modules/sandboxed-containers-building-blocks.adoc[leveloffset=+1]
 include::modules/sandboxed-containers-rhcos-extensions.adoc[leveloffset=+1]
+.Additional resources
+
+* xref:../post_installation_configuration/machine-configuration-tasks.adoc#rhcos-add-extensions_post-install-machine-configuration-tasks[Adding extensions to RHCOS]
+
 include::modules/security-compliance-nist.adoc[leveloffset=+1]
+include::modules/sandboxed-containers-deploying-osc-with-other-ocp-components.adoc[leveloffset=+1]
+.Additional resources
+
+* xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-creating-storage-class_virt-configuring-local-storage-for-vms[Configuring local storage for virtual machines]
```

Comment:
You should add an xref for [configuring live migration](https://docs.openshift.com/container-platform/4.9/virt/live_migration/virt-configuring-vmi-eviction-strategy.html) in VMs.